README.md

```diff
@@ -74,6 +74,17 @@ When multiple constraints are provided they will _all_ be satisfied.
 
 The cluster will be provisioned using the fastest available mechanism, because your development inner loop is a precious thing.
 
+## Provisioning topology
+
+In kraft cluster (the default) the extension will generate nodes using the `broker, controller` roles until it reaches the desired number of brokers or controllers (which ever is lowest) at which point it will continue deploying the remaining role.
```

Comment:
> In kraft cluster

Should we introduce some term like cluster mode [kraft, zookeeper]?

Reply:
Hmm,  my first draft locally I called it `kraft mode` but thought `a kraft cluster` aligned better with the use of the `KraftCluster` annotation. 

The mode terminology would be extension specific as Apache Kafka talks about KRaft clusters IIRC